services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java

```diff
@@ -109,6 +109,10 @@ private boolean isStatisticFile() {
     return s3Key.endsWith("stats") || s3Key.matches(".*local_stats_\\d");
   }
 
+  private boolean isCclConfigFile() {
+    return s3Key.endsWith("stats") || s3Key.matches(".*config-v\\d");
```

Comment:
why `endsWith("stats")` ???

```suggestion
    return s3Key.matches(".*config-v\\d");
```

Reply:
it is a mistake , thanks !